lib/src/widgets/movies_stream_builder.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'card_grid.dart';
+
+class MoviesStreamBuilder extends StatelessWidget {
+  final moviesBloc;
+
+  const MoviesStreamBuilder({Key key, this.moviesBloc}) : super(key: key);
+
+  Widget _buildContent() {
+    moviesBloc.fetchAllMovies();
```

Comment:
What do you think about leaving the call to fetchMovies outside the build function, mainly beacuse of a separation of concerns principle

Reply:
I made changes about this, i moved _buildContent of StreamBuilder on search_screen and only let responsability to create the gridview on this class (grid_builder).